Source/OIDAuthorizationRequest.m

```diff
@@ -126,11 +143,7 @@ - (instancetype)initWithConfiguration:(OIDServiceConfiguration *)configuration
     _scope = [scope copy];
     _redirectURL = [redirectURL copy];
     _responseType = [responseType copy];
-    // Attention: Please refer to https://github.com/openid/AppAuth-iOS/issues/105
-    // If you change the restriction on response type here, you must also update initWithCoder:
-    if (![_responseType isEqualToString:OIDResponseTypeCode]) {
-      // AppAuth only supports the `code` response type.
-      // Discussion: https://github.com/openid/AppAuth-iOS/issues/98
+    if (![self.class isSupportedResponseType:_responseType]) {
```

Comment:
Only properties should use dot notation per style guide `[self class]`, sorry I missed this earlier.

Other than that it's ready :)